creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -824,7 +830,7 @@ class SnapbackSM {
    *
    * Polls secondary for MaxSyncMonitoringDurationInMs
    */
-  async additionalSyncIsRequired (userWallet, primaryClockValue, secondaryUrl, syncType) {
+  async additionalSyncIsRequired (userWallet, primaryClockValue = -1, secondaryUrl, syncType) {
```

Comment:
hmm I feel like setting this as -1 can be problematic. if there actually is a falsy value for primaryClockValue, we probably should not assume there is no state and investigate as to why we are getting undefined?

Reply:
if a falsey value is passed in for primaryClockValue this default won't be used. this is only applied if no value is provided at all. seems fine to keep?

Reply:
I guess if primaryClockValue is `undefined`, it will be stored as -1. yeah it doesn't hurt to have